Add recipes for key-based caching (#7)
bitrise-io/workflow-recipes: recipes/android-key-cache-build-tasks.md

# (Android) Cache Gradle build tasks (Beta)

## Description

> Cache Gradle tasks with the new key-based caching steps, **Save Cache** and **Restore Cache**.

Reviewer: ```suggestion
Cache Gradle tasks with the new key-based caching Steps, **Save Cache** and **Restore Cache**.
```

## Prerequisites

> Make sure to read how to [cache Gradle dependencies](android-key-cache.md) and set up the workflow according to the guide. Caching build tasks is an opt-in feature that builds on caching Gradle dependencies.

Reviewer: ```suggestion
Make sure to read how to [cache Gradle dependencies](android-key-cache.md) and set up the Workflow according to the guide. Caching build tasks is an opt-in feature that builds on caching Gradle dependencies.
```

## Instructions

> [Gradle build cache](https://docs.gradle.org/current/userguide/build_cache.html) is a feature that persists the task outputs in the shared Gradle cache folder. Caching this folder in CI builds means that Gradle can reuse the task outputs from previous builds and can skip running the tasks when the inputs are unchanged.

Reviewer: ```suggestion
[Gradle build cache](https://docs.gradle.org/current/userguide/build_cache.html) is a feature that enables the storage of the task outputs in the shared Gradle cache folder. Caching this folder in CI builds means that Gradle can reuse the task outputs from previous builds and can skip running the tasks when the inputs are unchanged.
```

> This is an opt-in feature, there are two ways to enable build cache in a Gradle project:

Reviewer: ```suggestion
This is an opt-in feature. There are two ways to enable the build cache in a Gradle project:
```
